Source/WebKit/UIProcess/BrowsingContextGroup.h

```diff
@@ -44,6 +48,7 @@ class BrowsingContextGroup : public RefCounted<BrowsingContextGroup>, public Can
     ~BrowsingContextGroup();
 
     Ref<FrameProcess> ensureProcessForDomain(const WebCore::RegistrableDomain&, WebProcessProxy&, const WebPreferences&);
+    Ref<FrameProcess> processForConnection(IPC::Connection&, WebPageProxy&, const WebPreferences&);
```

Comment:
Should this be `ensureProcessForConnection()` because a `FrameProcess` may be created? Similar to `ensureProcessForDomain()` above.

Reply:
Yes.